hello-security/list-users.php

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Login</title>
+</head>
+
+<body>
+    <a href="index.php"> Take me home, country roads ... </a> <br>
+    <?php
+
+    session_start();
+
+    if (!isset($_SESSION["email"])) {
+        die("Only authenticated users are allowed");
+    }
+
+    include "db-connection.php";
+    $conn = openCon();
+
+    $table = "person";
+    $email = $_SESSION["email"];
+    $sql = "SELECT * from $table where email = :email;";
```

Comment:
My only concern is about using the table as variable. This is breaking the idea of the prepared statement as it allows any value to be passed to the sql for the table name. Currently this is not a problem in the code as nobody is changing the value of the table name. But what if somehow unintentionally the table name is changed to something else that we do not expect and do not want?